loops/task1.py

```diff
@@ -0,0 +1,7 @@
+numbers = [1, 2, 3, 5, 7, 8, 354, 456, 234, 654, 758, 324, 573, 543, 9, 10]
+divider = float(input('Enter any number as a divider: '))
+divided = []
+
+for i in numbers:
+    if i % divider == 0:
+        print(i)
```

Comment:
You should use formatting string all the time. In this task better to add all elements in the list and print their after loop.